src/ENVPAK.CPP

```diff
@@ -91,13 +102,15 @@ t }
 //=============================================================================
 WStr enExePath()		// full path to current executable
 {
-	char exePath[_MAX_PATH + 1];
 	WStr t;
+	#if !defined( ELIMINATE_WINDOWS)
+	char exePath[_MAX_PATH + 1];
 	if (GetModuleFileName(NULL, exePath, sizeof(exePath)) > 0)
```

Comment:
We should be able to write a function that gets the executable path on any platform. I believe I shared what EnergyPlus is doing for this.

Reply:
In order to use the EnergyPlus implementation, I need to include `windows.h`. Once I do that, I run into the same problem where some of `windows.h` definitions are included in CNGLOB.H.

Specifically lines [100](https://github.com/cse-sim/cse/blob/remove-windows-headers/src/CNGLOB.H#L100), [106](https://github.com/cse-sim/cse/blob/remove-windows-headers/src/CNGLOB.H#L106), and [107](https://github.com/cse-sim/cse/blob/remove-windows-headers/src/CNGLOB.H#L107).

Should I go ahead and change the definitions to CSE_BOOL and so on?

Reply:
Let's do this at the very end. I'm thinking that if we do a change, we might want to just use the standard data types (e.g., `bool`. 

@chipbarnaby Any thoughts?